### ISSUE TITLE
Attached image not exported if referenced from <img> tag

### ISSUE THREAD

**Issue by memeplex** [jupyter/nbconvert#1057]
I know of https://github.com/jupyter/nbconvert/issues/699. Nevertheless I find it confusing because I can easily export to html or reveal.js and the images are there as far as I reference them using `![]()` syntax. Now, as this is very limited in terms of formatting, many times I need to use an `<img>` tag instead, and then the attachments aren't exported to the output.

1. Take  a screenshot, copy & paste it into some notebook cell.
2. Download the notebook as html or reveal.js: ok, the image is still there.
3. Replace the markdown image syntax by the corresponding html syntax, referencing the same url (for example, "attachment:image.png").
4. Render the cell: ok, the image is still there.
5. Download the notebook as html or reveal.js: the image has disappeared! 

**Nbconvert version: 5.5.0**

**Comment by xiaohk** [comment]
Same issue here. I usually use `<img>` for better image formatting. Hopefully `nbconvert` can support it.

**Comment by xiaohk** [comment]
I solved it by adding the images in the correct path (relative to my `html` files) used in the `<img>` tags.

**Comment by MSeal** [comment]
Yes it likely is related. The way attachments got implemented in notebooks was fairly organic and this caused some difficult-to-fix issues for exporting to other formats. @xiaohk is the best work-around for now until that code path gets cleaned up (likely a 6.0 feature).

**Comment by axil** [comment]
I tried a lot of workarounds too, and also ended up with the same thing that @xiaohk described.

**Comment by wstomv** [comment]
> I solved it by adding the images in the correct path (relative to my `html` files) used in the `<img>` tags.

@xiaohk 
Can you explain how to do this, using a concrete example?

Does it apply to my situation, where I have a MarkDown cell with an attached image, shown via MarkDown syntax (not an HTML `<img>` tag; not an external image file).

I also tried `nbconvert` with an image that is embedded through `data:` and een encoded image. This also did not work for me.

**Comment by callegar** [comment]
Also the jupyter lab File->Print functionality is affected by this issue.

**Comment by jhgoebbert** [comment]
Often I need to resize images which I have inserted quick&easy with drag&drop to a notebook.  
This is (to my knowledge) from within JupyterLab only possible with `<img>` so that one ends up with:
```
<img src="attachment:9390596b-a5ba-4fdb-94b4-88995bc1e539.png" width="500"/>
``` 
GitHub can render notebooks with such elements, but nbconvert does not support it. An export of the notebook to PDF or HTML fails.

**Comment by i2000s** [comment]
Any update?
